app.rb

```diff
@@ -3,14 +3,36 @@
 require_relative 'book'
 require_relative 'rentals'
 require_relative 'manage_people'
+require 'json'
+require_relative 'create_book'
 
 class App
-  attr_accessor :books, :people, :rental
+  attr_accessor :books, :people, :rentals
 
   def initialize
-    @books = []
-    @people = []
-    @rentals = []
+    @books = load_books_data
+    @people = load_people_data
+    @rentals = load_rentals_data
+  end
+
+  def save_books_data
+    save_data(@books, './data/books.json')
+  end
+
+  def save_people_data
+    save_data(@people, './data/people.json')
+  end
+
+  def save_rentals_data
+    save_data(@rentals, './data/rentals.json')
+  end
```

Comment:
In an attempt to exit your application, it crashed. Your code specifies that the data should be preserved in 3 files within './data/fileName.json'. However, this DIR doesn't exist, neither does the files. Kindly create them and ensure that it works and ask for another review.

<img width="463" alt="image" src="https://github.com/emmanuelsteven/OOP-school-library/assets/27709832/7ebecefa-29b8-45d9-9ad1-2ca431336c1f">